packages/verifiable-credentials/src/credentials.ts

```diff
@@ -245,8 +245,8 @@ export default class Credentials {
 		return data
 	}
 
-	private getResolver(): any {
-		const resolver = vdaResolver.getResolver(DID_REGISTRY_ENDPOINT);
+	private static getResolver(didRegistryEndpoint: string): any {
+		const resolver = vdaResolver.getResolver(didRegistryEndpoint);
```

Comment:
You can't have a hardcoded `DID_REGISTRY_ENDPOINT`.

You will need to add a new variable `environmentType` that is an instance of `@verida/account :: EnvironmentType`.

You will need to update the `@verida/did-resolver` to accept an EnvironmentType OR an endpointURI and hardcode the approporaite endpointUri's for each environment type.

Reply:
Does this mean I should add a config that contains the respective `config.environments`?

so that the  `vdaResolver.getResolver(EnvironmentType)` accepts the current environments .

Reply:
@cmcWebCode40 Yes, you will need to add that config into `did-resolver` so that mapping can be used for other packages.

Reply:
alright noted